R/import_metadata_and_counts.R

```diff
@@ -94,10 +94,12 @@ import_metadata_counts <- function(
     select(metadata_tbl, .data$file_id_db) |> mutate(
       original_path = file.path(original_dir, basename(.data$file_id_db)),
       cpm_path = file.path(cache_dir, "cpm", basename(.data$file_id_db))
-    )
+    ) |>
+    distinct()
   
   # Generate cpm from counts
-  cli_alert_info("Generating cpm from {.path {metadata_tbl$file_id_db}}. ")
+  unique_file_ids <- unique(metadata_tbl$file_id_db)
+  cli_alert_info("Generating cpm from {.path {unique_file_ids}}. ")
```

Comment:
No need to repeat the distinct operation:
```suggestion
  cli_alert_info("Generating cpm from {.path {counts_path$file_id_db}}. ")
```